lib/contracts/shared-contracts.ts

```diff
@@ -10,6 +10,10 @@ export namespace SharedContracts {
         codename?: string;
         external_id?: string;
     }
+    export interface IWorkflowReferenceObjectContract {
```

Comment:
I wouldn't call it `IWorkflowReferenceObjectContract` as there might be other cases where a reference with just codename & id are required and this could be reused. Especially since it's located in `SharedContracts` namespace.

I would name it like `ICodenameIdReferenceContract`